designs/virtualization/4.5/Scheduling/Firmware-config/Firmware-config.md

```diff
@@ -0,0 +1,25 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# Firmware Config
```

Comment:
Is this something that should ultimately be documented until the "Scheduling" section of the documentation? Just wondering if this PR should extend that rather than be it's own section of documentation?

Reply:
We will merge it into one doc after the separate PRs get merged :)